src/components/meldekort/2-hovedseksjon/behandling/utfylling/MeldekortUtfylling.tsx

```diff
@@ -109,13 +168,88 @@ export const MeldekortUtfylling = ({ meldekortBehandling }: Props) => {
+                                            Teksten vises i vedtaksbrevet til bruker.
+                                        </BodyShort>
+
+                                        {meldekortBehandling.tekstTilVedtaksbrev !==
+                                            formContext.watch('tekstTilVedtaksbrev') && (
+                                            <HStack gap="1">
+                                                <InlineMessage status="warning">
+                                                    Teksten er endret og ikke lagret.
+                                                </InlineMessage>
+                                                <HelpText>
+                                                    Endringene blir lagret ved &apos;Lagre og
+                                                    beregn&apos;, og &apos;Send til beslutter&apos;
+                                                </HelpText>
+                                            </HStack>
+                                        )}
+                                    </VStack>
+                                }
+                                minRows={5}
+                                resize={'vertical'}
+                                value={field.value ?? ''}
+                                onChange={field.onChange}
+                            />
+                        )}
+                    />
+                    <Button
+                        className={styles.forhåndsvisBrevButton}
+                        type="button"
+                        variant="secondary"
+                        size="small"
+                        loading={forhåndsvisBrev.isMutating}
+                        onClick={() => {
+                            forhåndsvisBrev.trigger(
+                                {
+                                    tekstTilVedtaksbrev: formContext.getValues(
+                                        'tekstTilVedtaksbrev',
+                                    )
+                                        ? formContext.getValues('tekstTilVedtaksbrev')
+                                        : null,
+                                },
+                                {
+                                    onSuccess: (blob) => {
+                                        window.open(URL.createObjectURL(blob!));
+                                    },
+                                },
+                            );
+                        }}
+                    >
+                        Forhåndsvis brev
+                    </Button>
+                    {forhåndsvisBrev.error && (
+                        <Alert variant="error" size="small">
+                            <BodyShort>Feil ved forhåndsvisning av brev</BodyShort>
+                            <BodyShort>{`[${forhåndsvisBrev.error.status}] ${forhåndsvisBrev.error.message}`}</BodyShort>
+                        </Alert>
+                    )}
```

Comment:
Kan dette være en separat komponent som kan brukes i begge komponentene der vi trenger forhåndsvisningen?

Reply:
Jeg synes per tidspunkt at vi får ganske lite verdi av å gjøre det. Det vi egentlig er kanskje ute etter, er å ha en standard komponent med textarea + knapp - type "Brevinput" - som er generell nok til å abstrahere mye av logikken som vi har rundt. Så har vi også ønsker av når vi har lyst til å potensielt vise textarea og/eller forhåndsvis knappen. 

For å få til noe sånnt, så krever det å ta hånd om alle steder vi har en form for forhådnsvisning av brev / brevinput - og her har vi gjort det på litt forskjellige måter med nøstingen av komponent treet.